services/galley/postgresql-migrations/001_meetings.sql

```diff
@@ -0,0 +1,102 @@
+-- This file is part of the Wire Server implementation.
+--
+-- Copyright (C) 2025 Wire Swiss GmbH <opensource@wire.com>
+--
+-- This program is free software: you can redistribute it and/or modify it under
+-- the terms of the GNU Affero General Public License as published by the Free
+-- Software Foundation, either version 3 of the License, or (at your option) any
+-- later version.
+--
+-- This program is distributed in the hope that it will be useful, but WITHOUT
+-- ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+-- FOR A PARTICULAR PURPOSE. See the GNU Affero General Public License for more
+-- details.
+--
+-- You should have received a copy of the GNU Affero General Public License along
+-- with this program. If not, see <https://www.gnu.org/licenses/>.
+
+-- Migration: Add meetings table to PostgreSQL
+-- Description: Creates the meetings table with all required fields, indices, and constraints
+
+CREATE TABLE IF NOT EXISTS meetings (
+    -- Meeting identification
+    id uuid NOT NULL,
+    domain text NOT NULL,
+
+    -- Meeting metadata
+    title text NOT NULL,
+    creator uuid NOT NULL,
+    creator_domain text NOT NULL,
+
+    -- Scheduling information
+    start_date timestamptz NOT NULL,
+    end_date timestamptz NOT NULL,
+    recurrence jsonb,
```

Comment:
Not sure if we should store the recurrence untyped, like this. This is really an open question, not a critique.